src/common/services/session/session.service.spec.js

```diff
@@ -288,6 +295,7 @@ describe('session service', function () {
     describe('with \'IDENTIFIED\' role', () => {
       beforeEach(() => {
         $cookies.put(Sessions.role, cortexRole.identified)
+        $cookies.put(Sessions.profile, cortexRole.registered)
```

Comment:
This should be set to `cruProfile` I think, same with the below change.

Reply:
Good catch!